modules/integration/tests-common/admin-clients/src/main/java/org/wso2/appserver/integration/common/clients/EventProcessorAdminServiceClient.java

```diff
@@ -96,6 +97,16 @@ public int getExecutionPlanConfigurationCount()
         }
     }
 
+    public ExecutionPlanConfigurationDto getActiveExecutionPlanConfiguration(String executionPlanName)
+            throws RemoteException {
+        try {
+            return eventProcessorAdminServiceStub.getActiveExecutionPlanConfiguration(executionPlanName);
+        } catch (RemoteException e) {
+            log.error("RemoteException", e);
+            throw new RemoteException();
```

Comment:
@thiliA 
Please do not create a new exception without passing the cause of the previous exception. This results to a loss in information (i.e. the original issue will be lost). In this case, you can simply log and throw the same exception object. (No need to create a new exception)